idaes/models/unit_models/stream_scaler.py

```diff
@@ -0,0 +1,239 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+"""
+Unit model to adjust size of streams to represent, for example, a stream being split across several identical units,
+which are then all modeled as a single IDAES unit
+"""
+from functools import partial
+
+from pyomo.environ import (
+    Block,
+    PositiveReals,
+    units as pyunits,
+    Var,
+)
+from pyomo.network import Port
+from pyomo.common.config import ConfigBlock, ConfigValue, In
+
+from idaes.core import (
+    declare_process_block_class,
+    UnitModelBlockData,
+    useDefault,
+)
+from idaes.core.util.config import (
+    is_physical_parameter_block,
+)
+from idaes.core.base.var_like_expression import VarLikeExpression
+from idaes.core.util.tables import create_stream_table_dataframe
+import idaes.core.util.scaling as iscale
+import idaes.logger as idaeslog
+
+from idaes.models.unit_models.feed import FeedInitializer as StreamScalerInitializer
+
+__author__ = "Douglas Allan"
+
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+@declare_process_block_class("StreamScaler")
+class StreamScalerData(UnitModelBlockData):
+    """
+    Unit model to adjust size of streams to represent, for example, a stream being split across several identical units,
+    which are then all modeled as a single IDAES unit
+    """
+
+    default_initializer = StreamScalerInitializer
```

Comment:
I don't think this works here because our state block is named `inlet_block` instead of `properties`. That also indicates a lack of test coverage.

Renaming `inlet_block` to `properties` and `outlet_block` to `scaled_expressions` would work.